README.md

```diff
@@ -8,12 +8,26 @@ The protocol is meant to be RESTful, over HTTP.
 
 The specification is currently written in **OpenAPI 3.0** and can be pretty-visualized in the [github pages][tspGhPages].
 
-## HOW-TO
+## Current version
+
+The current version of specification is currently implemented and supportd in the [Trace Compass trace-server][tcServer] (reference implementation) and what is currently supported by the [tsp-typescript-client][tspClient].
+
+Swagger can be used to generate the API version implemented in Trace Compass trace-server (see [here](#gernerate-the-specification)).
+
+## Future version
+
+Some proposal for additional endpoints and features are documented in the `./API-proposed.yaml`. All the proposed changes are still not confirmed and can change. The pretty-visualized file can be found [here][apiProposed]. A diff of the current version and future version will show the differences.
```

Comment:
We should note also that such diffs will also show the ordering, which differs between swagger-core's and this manual yaml. Ordering also even differs between different runs of trace-server and swagger-core init.

Reply:
I took the effort to in this PR to have the same order and the diff will show only the additional, proposed changes. However, I re-generating the API can change orderings. I think we should make the effort to keep the diff that way in the future.

Reply:
I add a comment below the instructions about generating the API.